Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -972,7 +973,8 @@ namespace Multiplayer
 
         // Signal to session management when there are no remaining players in a dedicated server for potential cleanup
         // We avoid this for client server as the host itself is a user and non-transient dedicated servers
```

Comment:
```suggestion
        // We avoid this for client server as the host itself is a user and dedicated servers that do not terminate when all players have exited
```

If we're moving away from transient as a term, it'd be good to update this comment as well.